modules/multi-arch-creating-podplacmement-config.adoc

```diff
@@ -0,0 +1,52 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="multi-architecture-creating-podplacement-config_{context}"]
+
+= Creating a ClusterPodPlacementConfig custom resource
+
+After installing the Multiarch Tuning Operator, you must create a `ClusterPodPlacementConfig` custom resource (CR).
+
+The `ClusterPodPlacementConfig` CR is used to enable or disable the Multiarch Tuning Operator's pod placement operand.
+
+[discrete]
+== How to configure the ClusterPodPlacementConfig CR
+
+.Example `ClusterPodPlacementConfig` CR YAML file
+[source,yaml]
+----
+apiVersion: multiarch.openshift.io/v1alpha1
```

Comment:
```suggestion
apiVersion: multiarch.openshift.io/v1beta1
```

There is an ongoing change for this in https://github.com/openshift/multiarch-tuning-operator/pull/106

I think you can already change any `v1alpha1` instance to be `v1beta1`.

Reply:
updated as per the suggestion.